docs/articles/testkube-dashboard-executions.md

```diff
@@ -0,0 +1,73 @@
+# Testkube Executions
+
+A Testkube execution is an automated test (written in a language or syntax aligned with your preferred tools such as Cypress, JMeter, 
```

Comment:
please merge the first two paragraphs into this one: (feel free to make minor adjustements)

The Executions dashboard is the central repository for all executions that run in your **Environment**. A Testkube execution happens every time a Test Workflow is triggered by any of the available mechanisms (see [Triggering Workflows].. ). Once triggered, all executions will be available on this dashboard for tracking and analysis. Execution filtering together with Custom Views allows you to quickly find and work with the executions that you need, based on a specific tool, label, execution tag or runner (read more about filtering below).